### ISSUE TITLE
The `negativePositiveSignPlacement` option can be ignored in some cases

### ISSUE THREAD

**Issue by BobKnothe** [BobKnothe/autoNumeric#330]
function `correctPNegOption` incorrectly assigns the placement with the following conditions:

`$('.an').autoNumeric({ positiveSignSymbol: true });`

```
<input type='text' name="cc" id="cc" class='an' value="-8000" data-digit-group-separator='.' data-decimal-character=',' data-currency-symbol=" €" data-currency-symbol-placement="s" data-negative-positive-sign-placement='l' />
```

`data-negative-positive-sign-placement='l'`  is over written with the `'p'` value.

**Comment by AlexandreBonneau** [comment]
The `correctPNegOption()` function has been created to improve the user experience when the currency has its currency sign on the right hand side of the number (which is the case for most currencies).

I think the `correctPNegOption` function is correct, but assume that `negativePositiveSignPlacement` can be *"not set"* by the user, but since the default option for `negativePositiveSignPlacement` is `'l'`, you cannot know if it has been set or not.

Hence, I have a solution to keep the improved user experience when using the currency sign on the right hand side ; I propose that the default option for `negativePositiveSignPlacement` will now be `null`, and in the `correctPNegOption()` function, if `null` is detected, then we can update the `negativePositiveSignPlacement` option to `'l'` (or `'p'` for the above special case).

**Comment by BobKnothe** [comment]
The negativePositiveSignPlacement as I had originally written was in reference to the currencySymbolPlacement.

**Comment by AlexandreBonneau** [comment]
This has been fixed in PR #336.
